engine/runtime-compiler/src/main/scala/org/enso/compiler/pass/analyse/alias/Graph.scala

```diff
@@ -5,14 +5,17 @@ import org.enso.syntax.text.Debug
 import org.enso.compiler.pass.analyse.alias.Graph.{Occurrence, Scope}
 
 import java.util.UUID
+import scala.collection.immutable.HashMap
 import scala.collection.mutable
 import scala.reflect.ClassTag
 
 /** A graph containing aliasing information for a given root scope in Enso. */
 sealed class Graph extends Serializable {
-  var rootScope: Graph.Scope = new Graph.Scope()
-  var links: Set[Graph.Link] = Set()
-  var nextIdCounter          = 0
+  var rootScope: Graph.Scope                              = new Graph.Scope()
```

Comment:
If `initXyz` is better pattern than calling `g.rootScope_$eq(rootScope);` - then `rootScope` could be made private as well, possibly.

Reply:
I tried and looks like a lot more stuff depends on it being public. I tried to keep the PR succinct for now.